plonky_block_proof_gen/src/lib.rs

```diff
@@ -12,18 +12,20 @@
 //! internal STARK tables of the zkEVM.
 //!
 //! The default method contains an initial set of ranges for each table, that
-//! can be overriden at will by calling
+//! can be overridden at will by calling
 //! `ProverStateBuilder::set_foo_circuit_size` where `foo` is the name of the
 //! targeted table. At the moment, plonky2 zkEVM contains seven tables:
 //! `arithmetic`, `byte_packing`, `cpu`, `keccak`, `keccak_sponge`, `logic` and
 //! `memory`.
 //!
-//! ```rust
+//! ```no_run
```

Comment:
Added `no_run` since `build()` will actually start constructing the plonky2 state. :upside_down_face: